debian-live/build.sh

```diff
@@ -490,17 +490,81 @@ function installPrerequisites {
     fi
   done
 
+  downloadYq
+  downloadTrivy
+}
+
+function downloadYq {
   loginfo "${FUNCNAME[0]}" "Install yq"
+
   sudo curl --silent --location https://github.com/mikefarah/yq/releases/latest/download/yq_linux_amd64 --output /usr/local/bin/yq
   if [ "$?" -ne 0 ]; then
     logerror "${FUNCNAME[0]}" "yq download failed"
     exit 1
   fi
+
   sudo chmod 755 /usr/local/bin/yq
   if [ "$?" -ne 0 ]; then
     logerror "${FUNCNAME[0]}" "yq permission update failed"
     exit 1
   fi
+
+  loginfo "${FUNCNAME[0]}" "yq installation done"
+}
+
+function downloadTrivy {
```

Comment:
@Joe-Public i have added a `downloadTrivy` that checks if it is running on a Github runner...

Reply:
Great, I guess it's most important to have it for the official releases